galsim/interpolatedimage.py

```diff
@@ -550,6 +546,182 @@ def __setstate__(self, d):
                       _force_stepk=self._stepk, _force_maxk=self._maxk)
 
 
+class InterpolatedKImage(GSObject):
+    """A class describing non-parametric profiles specified by samples -- i.e., images -- of their
+    complex Fourier transform.
+
+    The InterpolatedKImage class is useful if you have a non-parametric description of the Fourier
+    transform of an object as a pair of Images -- one for the real part and one for the imaginary
+    part -- of the complex-valued profile that you wish to manipulate / transform using GSObject
+    methods such as shear(), magnify(), shift(), etc.  Note that neither real-space convolution nor
+    photon-shooting of InterpolatedKImages is currently implemented.  Please submit an issue at
+    http://github.com/GalSim-developers/GalSim/issues if you require either of these use cases.
+
+    The primary intended use case for InterpolatedKImage is to create ChromaticObjects from
+    PSF-deconvolved multi-band HST images, a functionality that will be added in a future
+    pull-request.
```

Comment:
This para is probably not necessary.  It doesn't really add anything to the understanding of InterpolatedKImage itself.